Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts

```diff
@@ -78,6 +91,24 @@ function publishEvent(feature, properties: any): void {
     }
 }
 
+export async function unzip(zipLocation: string, unzipLocation: string): Promise<void> {
```

Comment:
is it fine to use combination of async & promise?

Reply:
looks ok to me, same code is used for DownloadPackage task.